web/modules/custom/owner/owner.behat.inc

```diff
@@ -28,137 +29,70 @@ class OwnerSubContext extends DrupalSubContextBase implements DrupalSubContextIn
   protected $owners = [];
 
   /**
-   * Creates a person owner with data provided in a table.
+   * Creates one or many owners with data provided in a table.
    *
    * Table format:
-   * | name | Mightily Oats |
+   * | type                  | name             | id                           |
+   * | Company               | Mightily Oats    | http://example.com/some/path |
+   * | Local authority       | Gotham City Hall |                              |
+   * | Private Individual(s) | Jane Roe         |                              |
    *
-   * @param TableNode $person_table
-   *   The person table.
+   * @param TableNode $owners_table
+   *   The owners table.
    *
-   * @Given (the following )person:
-   */
-  public function givenPerson(TableNode $person_table) {
-    // Convert aliased field names.
-    $values = $this->convertFieldAliases('person', $person_table->getRowsHash());
-
-    $this->createOwner('person', $values);
-  }
-
-  /**
-   * Creates a number of person owners with data provided in a table.
+   * @throws \Behat\Mink\Exception\ExpectationException;
+   *   If a wrong type has been passed.
    *
-   * Table format:
-   * | name          |
-   * | Mightily Oats |
-   * | John Irwin    |
-   *
-   * @param TableNode $person_table
-   *   The person table.
-   *
-   * @Given (the following )people:
+   * @Given (the following )owner/owners:
    */
-  public function givenPeople(TableNode $person_table) {
-    foreach ($person_table->getColumnsHash() as $person) {
-      // Convert aliased field names.
-      $values = $this->convertFieldAliases('person', $person);
-
-      $this->createOwner('person', $values);
-    }
-  }
-
-  /**
-   * Creates a organisation owner with data provided in a table.
-   *
-   * Table format:
-   * | name | Red birds |
-   *
-   * @param TableNode $organisation_table
-   *   The organisation table.
-   *
-   * @Given (the following )organisation:
-   */
-  public function givenOrganisation(TableNode $organisation_table) {
-    // Convert aliased field names.
-    $values = $this->convertFieldAliases('organisation', $organisation_table->getRowsHash());
-
-    $this->createOwner('organisation', $values);
-  }
+  public function givenOwner(TableNode $owners_table) {
+    $term_storage = \Drupal::entityTypeManager()->getStorage('taxonomy_term');
+    foreach ($owners_table->getHash() as $row) {
+      // If no type was specified, fallback to 'Private Individual(s)'.
+      $row['type'] = isset($row['type']) ? $row['type'] : 'Private Individual(s)';
```

Comment:
Is this what happens in the UI as well? why do we need to fallback since the field instance does not include a default?